packages/core/src/codewhisperer/service/transformByQ/transformMavenHandler.ts

```diff
@@ -0,0 +1,209 @@
+/*!
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+import * as vscode from 'vscode'
+import { FolderInfo, transformByQState } from '../../models/model'
+import { getLogger } from '../../../shared/logger'
+import * as CodeWhispererConstants from '../../models/constants'
+import { spawnSync } from 'child_process' // Consider using ChildProcess once we finalize all spawnSync calls
+import { CodeTransformMavenBuildCommand, telemetry } from '../../../shared/telemetry/telemetry'
+import { codeTransformTelemetryState } from '../../../amazonqGumby/telemetry/codeTransformTelemetryState'
+import { MetadataResult } from '../../../shared/telemetry/telemetryClient'
+import { ToolkitError } from '../../../shared/errors'
+import { throwIfCancelled } from '../securityScanHandler'
```

Comment:
Not sure if this is the right `throwIfCancelled` to use?

Reply:
BLOCKER: This is the wrong import. Nice catch.